ztp/gitops-subscriptions/argocd/example/image-based-upgrades/example-oadp-policy.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: Placement
```

Comment:
This policy is using the Placement API rather than PlacementRule. How does this align with support in TALM and requirements on having ManagedClusterSets defined?

Reply:
TALM operates agnostically to the binding mechanisms of policies within clusters.

Reply:
That's good on the TALM side, but does this imply that the ManagedCluster is part of a ManagedClusterSet so that the binding works properly? Will a cluster installed through our normal ZTP process be a part of a ManagedClusterSet and thus the binding here will work as expected?

Reply:
Thanks @imiller0 for highlighting the `ManagedClusterSet`. We do not need to create one explicitly as we can bind the `common` namespace to the default provided `global` `ManagedClusterSet`. However, a `ManagedClusterSetBinding` needs to be created for that. I have updated the example policy to also include an example of this binding.